catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -108,8 +107,6 @@
 
     private final OperationsStorageSupport opsStorageSupport;
 
-    private Historian historian;
```

Comment:
This was left in from a previous PR where it was removed, and was not being used anymore.